src/main/java/service/GenericService.java

```diff
@@ -0,0 +1,11 @@
+package service;
+
+import java.util.List;
+
+public interface GenericService<T>{
```

Comment:
you already have an interface with the same methods : GenericDAO. Then why do you need an interface here?

Reply:
Actually, I don't need it. I'm working on my project. I'll request you to re-review.